test-definitions/fern/exhaustive/definition/endpoints/params.yml

```diff
@@ -0,0 +1,59 @@
+service: 
+  auth: true
+  base-path: /params
+  endpoints: 
+
+    getWithPath: 
+      docs: GET with path param
+      path: /path/{param}
+      path-parameters: 
+        param: string
+      method: GET
+      response: string
+    
+    getWithQuery: 
+      docs: GET with query param
+      path: ""
+      method: GET
+      request: 
+        name: GetWithQuery
+        query-parameters: 
+          query: string #mandatory for test 
+          number: integer
+
+    getWithAllowMultipleQuery: 
+      docs: GET with multiple of same query param 
+      path: ""
+      method: GET
+      request: 
+        name: GetWithMultipleQuery
+        query-parameters: 
+          query: 
+            type: string
+            allow-multiple: true
+          numer: 
+            type: integer
+            allow-multiple: true
+
+    getWithPathAndQuery: 
+      docs: GET with path and query params
+      path: /path/{param}
+      path-parameters: 
+        param: string
+      method: GET
+      request: 
+        name: GetWithPathAndQuery
+        query-parameters: 
+          query: string #mandatory for test
+
+
+    modifyWithPath: 
+      docs: PUT to update with path param 
+      path: /path/{param}
+      path-parameters: 
+        param: string
+      method: PUT
+      request: 
+        name: ModifyResourceAtPath
+        body: string
+      response: string
```

Comment:
@sahil485 can we move `inlined-requests`, `no-req-body` and `req-with-headers` to the root level under the `/definition` folder